external/kafka-0-10-token-provider/src/test/scala/org/apache/spark/kafka010/KafkaTokenUtilSuite.scala

```diff
@@ -233,8 +233,27 @@ class KafkaTokenUtilSuite extends SparkFunSuite with KafkaDelegationTokenTest {
     assert(jaasParams.contains(tokenPassword1))
   }
 
-  test("isConnectorUsingCurrentToken without security should return true") {
-    val kafkaParams = Map[String, Object]().asJava
+  test("isConnectorUsingCurrentToken without security credentials enabled should return true") {
```

Comment:
true -> None

Reply:
I see the potential from understanding point if view but don't see the immediate need to differentiate between the 3 states: `true`, `false` and no token. It would add more lines and complexity without actual value. `true` means no action needed which fits for the usage now.

Reply:
Ah I confused with the parameter. My bad.

Reply:
OK, thanks.